test/test_force.js

```diff
@@ -111,263 +104,119 @@ function main(args) {
-    log('Updating cordova plugin at ' + branch, COLOR.green);
-    runProcessThrowError(path.join('tools', 'update.sh') + ' -b ' + branch + ' -o ' + os, pluginRepoDir);
-}
-
-//
-// Update forceios/forcedroid script to use local plugin repo
-//
-function editForceScriptToUseLocalPluginRepo(tmpDir, os) {
-    log('Editing  ' + forcePackageNameForOs(os) + '.js to use local cordova plugin', COLOR.green);
-    miscUtils.replaceTextInFile(path.join(tmpDir, 'node_modules', forcePackageNameForOs(os), 'node', forcePackageNameForOs(os) + '.js'), new RegExp('\'cordova plugin add .*\'', 'g'), '\'cordova plugin add ../SalesforceMobileSDK-CordovaPlugin\'');
-}
-
-//
-// Update createApp.sh to not do pod install
-// 
-function editCreateAppToNotDoPodInstall(tmpDir) {
-    log('Editing  createApp.sh to not do pod install', COLOR.green);
-    miscUtils.replaceTextInFile(path.join(tmpDir, 'node_modules', 'forceios', 'build', 'app_template_files', 'createApp.sh'), new RegExp('^\\s*pod install', 'g'), '# pod install');
-
-}
-
-
-//
-// Update podfile to use local ios repo
-// 
-function editPodfileToUseLocalRepo(appDir) {
-    log('Editing podfile to use local ios repo', COLOR.green);
-    miscUtils.replaceTextInFile(path.join(appDir, 'Podfile'), new RegExp('pod (\'Salesforce.*\')', 'g'), 'pod $1, :path => \'../SalesforceMobileSDK-iOS\'');
-    miscUtils.replaceTextInFile(path.join(appDir, 'Podfile'), new RegExp('pod (\'Smart.*\')', 'g'), 'pod $1, :path => \'../SalesforceMobileSDK-iOS\'')
-}
-
-//
-// Helper to run arbitrary shell command - errors caught (and reported)
-// Returns true if successful
-//
-function runProcessCatchError(cmd, msg, dir) {
-    var success = false;
-    log('Running: ' + cmd);
-    if (dir) shelljs.pushd(dir);
-    try {
-        execSync(cmd);
-        if (msg) log('!SUCCESS! ' + msg, COLOR.yellow);
-        success = true;
-    } catch (err) {
-        if (msg) log('!FAILURE! ' + msg, COLOR.red);
-        console.error(err.stderr.toString());
-    }
-    finally {
-        if (dir) shelljs.popd();
-        return success;
-    }
-}
-
-//
-// Helper to run arbitrary shell command - errors thrown
-//
-function runProcessThrowError(cmd, dir) {
-    log('Running: ' + cmd);
-    if (dir) shelljs.pushd(dir);
-    try {
-        execSync(cmd);
-    }
-    finally {
-        if (dir) shelljs.popd();
-    }
-}
-
-//
-// Print important information
-//
-function log(msg, color) {
-    if (color) {
-        console.log(color + msg + COLOR.reset);
-    }
-    else {
-        console.log(msg);
-    }
-}
-
-
-//
-// Return random number between n/10 and n
-//
-function random(n) {
-    return (n/10)+Math.floor(Math.random()*(9*n/10));
-}
-
-//
 // Helper to validate version
 // 
 function validateVersion(version) {
     if (version.match(/\d\.\d\.\d/) == null) {
-            log('Invalid version: ' + version, COLOR.red);
-            process.exit(1);
+        utilslog('Invalid version: ' + version, COLOR.red);
```

Comment:
Indeed! Actually the whole method can go. The version is now pulled from shared/constants.js.